src/documents/APIChangesFrom1_5To2_0.md

```diff
@@ -10,6 +10,59 @@ This page contains a list of the features, APIs, and etc. that have changed in I
 
 ### Preferences Framework ###
 
+#### Panel Changes ####
+Changes and additions are being made to UI Options in order to improve its compliance with accesibility standards.
+
+##### Message Bundle Keys #####
+
+######Additions######
+* `textSizeDescr: {messagekey: "textSizeDescr"}`
+* `textFontDescr: {messagekey: "textFontDescr"}`
+* `lineSpaceDescr: {messagekey: "lineSpaceDescr"}`
+* `contrastDescr: {messagekey: "contrastDescr"}`
+
+######Changes######
+<table>
+<tr><th> 1.5</th><th>2.0</th></tr>
+<tr>
+    <td><code>choiceLabel: {messagekey: "speakChoiceLabel"}</code></td>
+    <td><code>speakDescr: {messagekey: "speakDescr"}</code></td>
+</tr>
+<tr>
+    <td><code>choiceLabel: {messagekey: "tocChoiceLabel"}</code></td>
+    <td><code>tocDescr: {messagekey: "tocDescr"}</code></td>
+</tr>
+<tr>
+    <td><code>choiceLabel: {messagekey: "simplifyChoiceLabel"}</code></td>
+    <td><code>simplifyDescr: {messagekey: "simplifyDescr"}</code></td>
```

Comment:
I don't think it is necessary to show the `{messagekey: "..."}` in the changes and additions for the message bundles section. The `messagekey` is specific to the prototree. The messages could be accessed through the prototree, msgLookup and etc. I think simply listing the keys here would be fine (same in the Additions section. You could however consider listing any changes to the text in the default bundles, but I'm not really sure that is necessary either.